vue-components/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+	plugins: [
+		require( 'autoprefixer' ),
```

Comment:
I find it a bit iffy to never directly depend on autoprefixer or postcss-autoprefixer and yet put it to work here, but it is the same way in bridge and termbox and works. Fair.

It also totally makes sense to have it so this patch mentioning is just making explicit what happened implicitly through vue-cli already and adding another plugin next to it.